Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -32,13 +32,37 @@ struct PitchSpeller {
             UnorderedPair(.down,.down)
         ]
     )
-
-    static let adjacencyCarrying = AdjacencyCarrying.build(from: PitchSpeller.tendencyGraph)
-    static let tendencyMask: AdjacencyCarrying<Graph<FlowNode<Cross<Int,Tendency>>>> = adjacencyCarrying.pullback { node in
+    
+    static let eightTendencyLink: [(Pitch.Class, Tendency)] = [
+        (.init(value: 0), .down),
+        (.init(value: 1), .up),
+        (.init(value: 3), .down),
+        (.init(value: 4), .up),
+        (.init(value: 5), .down),
+        (.init(value: 6), .up),
+        (.init(value: 7), .down),
+        (.init(value: 8), .up),
+        (.init(value: 9), .up),
+        (.init(value: 10), .down),
+        (.init(value: 11), .up),
+        ]
+    
+    static let eightLookup = Set<UnorderedPair<Cross<Pitch.Class, Tendency>>> (
+        PitchSpeller.eightTendencyLink.lazy
```

Comment:
Once you declare `.lazy` once, the `.map`s should compose into [`LazyMapSequence`](https://developer.apple.com/documentation/swift/lazymapsequence).

I think only a singly `.lazy` is needed. Could be verified, though.

Reply:
hah, is `lazy` a monad ? ;)